ledger-service/jwt/src/main/scala/com/digitalasset/jwt/jwtLeeway.scala

```diff
@@ -7,8 +7,8 @@ import com.auth0.jwt.JWT
 import com.auth0.jwt.algorithms.Algorithm
 import com.auth0.jwt.interfaces.{Verification}
 
-final case class LeewayOptions(
-    leeway: Option[Long] = None,
+final case class JwtTimestampLeeway(
```

Comment:
Please change the file name to match the class name (including the capitalization of J